src/views/partials/landing.js

```diff
@@ -105,5 +105,6 @@ export const landing = () => `
 <section class='see-if-data-breach' data-enter-transition>
   <h2>${getMessage('see-if-data-breach')}</h2>
   <a class='button primary' data-cta-id='landing-2' href='/user/breaches'>${getMessage('get-started')}</a>
+  <p class='hibp-attribution'>${getMessage('hibp-footer-attribution')}</p>
```

Comment:
I may be stretching semantic HTML. Could this be a [`<cite>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite) tag?

Reply:
_Note: You may have to update your CSS to counteract default styling on this tag._ 

```suggestion
  <cite class='hibp-attribution'>${getMessage('hibp-footer-attribution')}</cite>
```

Reply:
If you do switch to `cite` not sure if you also need that wrapper `<p>` tag still since I think both `<a>` and `<cite>` tags are inline and not block elements, so it might put them side by side and not stacked. But that's between y'all and the UX gods.